Makefile

```diff
@@ -3,7 +3,7 @@
 test: flake8 pylint pytest
 
 pylint:
-	pylint sqlalchemydiff -E
+	pylint sqlalchemydiff -E --disable=E1102
```

Comment:
without:

> ************* Module sqlalchemydiff.comparer
E:137,22: inspector.get_table_names is not callable (not-callable)
E:200, 8: ignore_manager.get is not callable (not-callable)
E:207, 8: ignore_manager.get is not callable (not-callable)
E:214, 8: ignore_manager.get is not callable (not-callable)
E:221, 8: ignore_manager.get is not callable (not-callable)
E:228, 8: ignore_manager.get is not callable (not-callable)
E:289,11: inspector.get_foreign_keys is not callable (not-callable)
E:309,11: inspector.get_primary_keys is not callable (not-callable)
E:327,11: inspector.get_indexes is not callable (not-callable)
E:349,11: inspector.get_columns is not callable (not-callable)
E:373,15: inspector.get_check_constraints is not callable (not-callable)
E:394,15: inspector.get_enums is not callable (not-callable)
E:420,11: type_.compile is not callable (not-callable)
E:448,30: errors['tables_data'].setdefault is not callable (not-callable)
E:449,20: table_d.setdefault is not callable (not-callable)
************* Module sqlalchemydiff.util
E: 71,12: stream.write is not callable (not-callable)
E: 98,14: uri.rsplit is not callable (not-callable)
E:106, 4: sqlalchemy_base.metadata.create_all is not callable (not-callable)
E:124,27: data.strip is not callable (not-callable)
E:138,15: data.count is not callable (not-callable)
E:145,15: data.split is not callable (not-callable)
E:152,16: item.strip is not callable (not-callable)
E:152,41: data.split is not callable (not-callable)